rust_src/src/alloc.rs

```diff
@@ -112,4 +114,20 @@ pub fn purecopy(obj: LispObject) -> LispObject {
     }
 }
 
+/// Return a newly allocated marker which does not point to any place.
+#[lisp_fn(c_name = "make_marker", name = "make-marker")]
+pub fn make_marker_lisp() -> LispObject {
+    let val: LispObject = unsafe { allocate_misc(Lisp_Misc_Type::Lisp_Misc_Marker) };
+    let mut p: LispMarkerRef = val.as_marker().unwrap();
+
+    // Set the properties of the marker to nothing
+    p.set_buffer(ptr::null_mut());
+    p.set_charpos(0 as isize);
+    p.set_bytepos(0 as isize);
```

Comment:
The `as isize` can be removed. (If I'm mistaken and it gives an error if you don't specify the type, then it's more correct to write `0isize`)

Reply:
Thank you, I will take care of the changes shortly!